### PR TITLE
Harry whorlow

### PR DESCRIPTION
**Pull request by harry-whorlow**
Scroll to bottom of page, usefully for viewing footers and sub-pages content.

### CODE REVIEW

**Comment by blesuer0729** [comment]
The files in the second commit (scrollBottom.txt, scrollBottom.png, test.html) probably don't belong in the base directory of the repo

**Comment by harry-whorlow** [comment]
> The files in the second commit (scrollBottom.txt, scrollBottom.png, test.html) probably don't belong in the base directory of the repo

where do you want the docs located?

**Comment by blesuer0729** [comment]
I'm working on some changes right now that add a README under every module so that we can have somewhere to put the docs. I'll let you know when it's up then you can modify your docs accordingly

**Comment by blesuer0729** [comment]
check [these changes](https://github.com/oliverisMisiunas/jslim-library/pull/5#issue-1179668005)

**Comment by blesuer0729** [comment]
move your code to a new directory `/modules/ScrollToBottom/ScrollToBottom.js` and then update your import statement for the new path. Add your documentation in the new directory as a **README** using the same structure the other modules use.

**Comment by oliverisMisiunas** [comment]
Hey, Harry! Thank you very much for your contribution. We are resolving one little issue with Brennan (blesuer0729) which will end up changing the project structure a little bit. I will let you know immediately when it's ready so you could resolve the conflict that isn't letting me merge your PR.

**Comment by harry-whorlow** [comment]
> Hey, Harry! Thank you very much for your contribution. We are resolving one little issue with Brennan (blesuer0729) which will end up changing the project structure a little bit. I will let you know immediately when it's ready so you could resolve the conflict that isn't letting me merge your PR.

No worries, I'm looking to add a couple more modules for making hamburger Menus easier to set up. I'm still thinking through how to make that work with multiple instances, but should be able to push it later today or tomorrow.

**Comment by oliverisMisiunas** [comment]
Hey, once again! @blesuer0729 has improved the file structure so it has changed a bit. Once you're ready, you can take a look at the new structure and submit another PR. I am sorry for the inconvenience that was caused earlier. If the structure is a bit unclear you can take a look at the updated tutorial here: https://www.jslim.org/howtoopensource

Good luck!

File: src/plugin/JSlim.js [hunk 1]
```diff
@@ -9,29 +9,31 @@ import ScrollToPos from "../modules/ScrollToPos.js";
 import ScrollToTop from "../modules/ScrollToTop.js";
 import ToggleOnClick from "../modules/ToggleOnClick.js";
 import ToggleOnScroll from "../modules/ToggleOnScroll.js";
-
+import ScrollToBottom from "../modules/ScrollToBottom.js";
```
Comment (blesuer0729) [inline src/plugin/JSlim.js:12]: ScrollToBottom should actually be at ../modules/ScrollToBottom/ScrollToBottom.js

Make a directory called ScrollToBottom/ then add your code and  a new file called README.md with a description and example of what the code should do.

**Comment by blesuer0729** [comment]
You should go to your forked repo and make sure you click the 'Fetch upstream' button so that your project mirrors the latest file structure. Then you won't have merge conflicts.

**Comment by harry-whorlow** [comment]
Third times the charm, hopefully this resolves the issues.

**Comment by oliverisMisiunas** [comment]
Awesome! Thank you very much for contributing! I am merging your PR now and will update the docs/mention page after. I really appreciate your contribution, very cool!

**Pull request merged**